app/views/partners/approve_partner.html.erb

```diff
@@ -36,7 +36,7 @@
           <p>Name: <%= @agency[:name] %></p>
           <p>Distributor Type <%= @agency[:distributor_type] %></p>
           <p>Agency Type: <% @agency[:agency_type] %></p>
-          <p>Proof of Agency Status (Link) <%= @agency[:proof_of_agency_status] %></p>
+          <p>Proof of Agency Status (Link): <%= link_to File.basename(@agency[:proof_of_agency_status]), documentation_url(@agency[:proof_of_agency_status]) %></p>
```

Comment:
Looks great, one minor change: can you open it in a new window? I think you just need to add the `rel: 'nofreferrer'` to your link_to `<%= link_to File.basename(@agency[:proof_of_agency_status]), documentation_url(@agency[:proof_of_agency_status]), rel: 'noreferrer' %>`

Reply:
ok. done.